cloud-operator/templates/cloud-operator-deployment.yaml

```diff
@@ -88,6 +88,8 @@ spec:
               value: "{{ .Values.env.streamSuccessPeriods.connect }}"
             - name: STREAM_SUCCESS_PERIOD_AUTH
               value: "{{ .Values.env.streamSuccessPeriods.auth }}"
+            - name: HTTPS_PROXY
+              value: "{{ .Values.env.httpsProxy }}"
```

Comment:
Why did you remove it from `values.yaml`???
Add it back. Every single Helm value should be listed in `values.yaml`, to serve as documentation and to make it clear what the default value is.